src/views/CardStack/CardStackTransitioner.js

```diff
@@ -84,11 +89,27 @@ class CardStackTransitioner extends React.Component<Props> {
     return transitionSpec;
   };
 
+  _getMode() {
+    const { navigation, mode, router } = this.props;
+    let screenOptions;
+    if (
+      this.props.navigation.state.routes &&
+      this.props.navigation.state.routes[1]
+    ) {
+      router.getScreenOptions(
```

Comment:
Shouldn't we be storing this on `screenOptions`?

Reply:
Yes, I fixed it to make more sense.